includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-courses.php

```diff
@@ -67,11 +68,23 @@ public function get_items( array $filters ): array {
 		if ( 'count_of_completions' === $course_args['orderby'] ) {
 			add_filter( 'posts_orderby', array( $this, 'add_orderby_custom_field_to_query' ), 10, 2 );
 		}
+		$course_args   = apply_filters( 'sensei_analysis_overview_filter_courses', $course_args );
+		$courses_query = new WP_Query( $course_args );
 
-		$courses_query = new WP_Query( apply_filters( 'sensei_analysis_overview_filter_courses', $course_args ) );
+		remove_filter( 'posts_orderby', array( $this, 'add_orderby_custom_field_to_query' ), 10, 2 );
+
+		$all_courses_query  = new WP_Query(
+			array_merge(
+				$course_args,
+				[
+					'posts_per_page' => -1,
+					'fields'         => 'ids',
+				]
+			)
+		);
+		$this->all_item_ids = $all_courses_query->posts;
```

Comment:
Should this be reverted now?

Reply:
Hey I reverted it but as a change, I didn't use git revert since this change is upon the other change and it would be too big of a mess reverting multiple things a top of one another
[7b0a333ac8bdfe493fa3152cc7ce04f3ee7210d1](https://github.com/Automattic/sensei/pull/5091/commits/7b0a333ac8bdfe493fa3152cc7ce04f3ee7210d1)